### PR TITLE
Simplification of externals framework

### PR DESCRIPTION
**Pull request by jasongrout**
Building on @blink1073's work, I'm working on simplifying the externals framework even more.

### CODE REVIEW

**Comment by jasongrout** [comment]
FYI, I'm working on this in conjunction with updating ipywidgets at https://github.com/ipython/ipywidgets/pull/727

**Comment by blink1073** [comment]
I'm not going to look at this closely until Monday lest my head explodes ;)

**Comment by jasongrout** [comment]
@blink1073 - if it's quick, could you look at this and review it? I think it's a strict improvement, though I realize we might head in a different direction soon.

File: scripts/extension_helpers.js [hunk 1]
```diff
@@ -64,49 +79,16 @@ function createShimHandler(pkgName) {
       if (path.indexOf('/') === 0) {
         path = path.slice(1);
       }
-      path = 'var jupyter.externals["' + pkgName + '"]["' + path + '"]';
-      return callback(null, path);
+      var shim = 'var jupyter.externals["' + pkgName + '"]["' + path + '"]';
+      // console.info(request + ' translated to '+shim);
```
Comment (blink1073) [inline scripts/extension_helpers.js]: Remove comment?

**Comment by blink1073** [comment]
Can you also pin the codemirror version to 5.17.0 so we get a passing build?

**Comment by blink1073** [comment]
👍 Thanks!

**Pull request merged**